Cargo.toml

```diff
@@ -153,6 +153,12 @@ debug-assertions = true
 inherits = "release"
 debug-assertions = false
 
+[profile.production-stripped]
+inherits = "production"
+strip = true
+lto = "thin"
+codegen-units = 1
```

Comment:
Hrm. Perhaps it make sense to simply make the production profile have these properties? The only thing here I have doubts about is `strip`, which would make traces from nightlies a bit useless. Perhaps that can be exposed an an option somehow?

Reply:
Would it make sense to rename the current `production ` to `nightly` or `production-unstripped` so that the new `production` profile is the stripped version? In this case I think the nightly CI workflow will need to be updated.

Reply:
> Perhaps it make sense to simply make the production profile have these properties?

From my point of view it would make sense to add the `lto` and `codegen-units` options to `production`. I was just not sure if people might be concerned about increased memory usage (during link-time when using `production` )

> Perhaps that can be exposed an an option somehow?

If `production-stripped` would differ by the `stripped` option, then the name would be fitting, and it would be "like an option" I guess.

Reply:
> Would it make sense to rename the current production to nightly or production-unstripped so that the new production profile is the stripped version?

I think in many cases you probably do want the debug information to get useful backtraces. My motivation is mainly to easily build a stripped version for benchmarks, since the `.so` size influences the time it takes until the first page is rendered.

Reply:
> since the `.so` size influences the time it takes until the first page is rendered.

Interesting, do you have any measurements?